spec/features/super_admin/add_custom_organisation_spec.rb

```diff
@@ -23,11 +23,18 @@
     end
 
     context 'when the organisation name is blank' do
-      it 'will error with the reason' do
+      it 'will error if its blank' do
         fill_in 'custom_organisations[name]', with: ' '
         click_on 'Allow organisation'
         expect(page).to have_content("Name can't be blank")
       end
+
+      it 'will error if its already in our register' do
+        CustomOrganisationName.create(name: 'Custom Org name')
+        fill_in 'custom_organisations[name]', with: 'Custom Org name'
+        click_on 'Allow organisation'
+        expect(page).to have_content("Name is already in our register")
```

Comment:
Are we referring to this as our register?

Maybe it can just say "This organisation already exists" ?

Reply:
Yeah to our register, I feel it gives Steve move context by saying "Name is already in our register"

Reply:
I mean just the naming here in general.
If the error message refers to it as our register, why doesn't the code do the same?